### DISCUSSION TITLE
Due date serious bug. It saves it to day before.

### DISCUSSION THREAD

**Discussion by whisperdancer** [TechbeeAT/jtxBoard#636] [General]
Version 2.04.04.ose
Android. Installed through f-droid

I wondered why all of a sudden I wasn't receiving task notifications. Then I discovered the issue is a bug with the due date. When I create a new task and set the due date, the app subtracts a day. Steps to reproduce:

Create new task
Tap Due field
Enter June 19 (Monday) and time 2pm
Tap Ok
The app saves it as June 18th (Sunday)

So if I want to enter a task due on June 19, I have to enter a due date of June 20th. I have to add a day to the due date to get the day I want. 

I'm curious if others are experiencing this also.

Thank you.

**Comment by AeliusSaionji** [comment]
I also have this issue


https://github.com/TechbeeAT/jtxBoard/assets/4342746/632f32ef-35cb-4d6e-ae91-eeccea774ae1

**Reply by patrickunterwegs** [reply]
Hello @AeliusSaionji , just to be sure, can you confirm that the issue is also fixed on your side?

**Comment by patrickunterwegs** [comment]
Hi @whisperdancer , hi @AeliusSaionji , 
thank you, the issue is related to some timing and timezone-related handling. So not everyone would experience that issue depending in which timezone you are. A new version is on the way that hopefully will fix this issue in all constellations.

**Comment by tdido** [comment]
Not sure it's the same as OP's case, but in my case with DAVx5 and Thunderbird, unless I manually specify the time zone for the task, it will get a "Local time" timezone in Thunderbird which is effectively UTC.

Has the fix you mentioned been implemented already? I'm on v2.05.06.ose.

Another way to deal with this for me would be to be able to se a default timezone for tasks.

**Reply by patrickunterwegs** [reply]
Hi @tdido , sorry for the late reply. Your issue is unrelated to the bug above. I think it is more an issue in Thunderbird, isn't it? But anyway, as you can already select a default time for new tasks, it makes sense to also offer the option to set a default timezone. I'll add it as a feature request.
